core/src/main/java/com/linecorp/armeria/server/ServerSslContextUtil.java

```diff
@@ -100,7 +101,7 @@ static SslContext buildSslContext(
             Iterable<? extends Consumer<? super SslContextBuilder>> tlsCustomizers) {
         return SslContextUtil
                 .createSslContext(sslContextBuilderSupplier,
-                        /* forceHttp1 */ false, tlsAllowUnsafeCiphers, tlsCustomizers);
+                        /* forceHttp1 */ false, tlsAllowUnsafeCiphers, Flags.useOpenSsl(), tlsCustomizers);
```

Comment:
Should we also add `useOpenSsl()` to `ServerBuilder`?

Reply:
I think that's a great idea. It might seem natural for the users to have both ways to configure `useOpenSsl()` on the server and the client side.

Reply:
@ikhoon I've added `useOpenSsl()` in the `ServerBuilder`. I'm not sure if it's good decision to implement `ServerSslContextUtil`'s `static SSLSession validateSslContext(SslContext sslContext)` method like below. 
- Check whether server uses openSsl by checking `sslContext`'s class 
```java 
 static boolean serverUsesOpenSsl(SslContext sslContext) {
    return sslContext instanceof OpenSslServerContext;
}
```